server/channel_chat.go

```diff
@@ -859,6 +859,19 @@ func (server *ChannelServer) gmCommand(conn mnet.Client, msg string) {
 		inst, err := field.GetInstance(plr.InstanceID())
 		pool := inst.DropPool()
 		pool.RemoveDrop(false, id)
+	case "npco":
```

Comment:
I think there is a system to override npc script behaviour e.g. cody not sure how it works in the slightest at the moment.

Reply:
I believe this is controlled in the string node of the wz file. Possibly leveraging the data in that node can be used to quickly create GMS-like scripts as well?

Reply:
I have had a look at that and I think the nodes under `/String/Npc.img/` are responsible for the bubble dialogue above an npc and default text that you get when double clicking them.

Reply:
Right. I think I may have misunderstood the behavior you are describing.